test/unit/specs/store/ledger.spec.js

```diff
@@ -123,13 +129,13 @@ describe(`Module: Ledger`, () => {
         })
 
         it(`fails if one of the function throws`, async () => {
-          dispatch = jest.fn(async () => Promise.reject(`error`))
+          dispatch = jest.fn(async () => Promise.reject(new Error(`error`)))
```

Comment:
Probably here there are some other occurencies of `jest.fn(async` ... `)` that can be dropped until you are not expecting something from `dispatch` using a simple `()` will work